pages/application-green.html

```diff
@@ -4,6 +4,76 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+    <link rel="stylesheet" href="../css/normalize.css" />
+    <link rel="stylesheet" href="../css/styles.css" />
```

Comment:
when you link the main styles, I would suggest using helper classes as well as it helps to significantly reduce the code